### PR TITLE
trailing slash documentation fix

### LINKED ISSUE #42: Wrong description for removeTrailingSlash()
In the https://finsweet.com/open-source/ts-utils/helpers doc the description for removeTrailingSlash()
Says: Compares if two arrays have the same exact values.
It should say something like: Removes the trailing slash from a given string.

### PR DESCRIPTION
**Pull request by ayezinzu**
Closes #42

### CODE REVIEW

**Review by alexiglesias93 — CHANGES_REQUESTED** [review]
Thank you!
Please:
- Add a changeset
- Reference the #42 issue in the PR

File: package.json [hunk 1]
```diff
@@ -57,5 +57,8 @@
     "typescript": "^4.9.5",
     "vitepress": "1.0.0-alpha.50",
     "vue": "^3.2.47"
+  },
+  "dependencies": {
+    "changeset": "^0.2.6"
```
Comment (alexiglesias93) [inline package.json]: unrelated dependency, please remove

**Review by alexiglesias93 — CHANGES_REQUESTED** [review]

**Review by alexiglesias93 — CHANGES_REQUESTED** [review]
CI is failing

**Comment by ayezinzu** [comment]
@alexiglesias93 Issues resolved

**Review by alexiglesias93 — CHANGES_REQUESTED** [review]
No updates to `.npmrc` are required...
Please get rid of it and the `pnpm-lock.yaml` updates.
Just run `pnpm install`

**Comment by ayezinzu** [comment]
> No updates to `.npmrc` are required... Please get rid of it and the `pnpm-lock.yaml` updates. Just run `pnpm install`

image

@alexiglesias93 Doing so causes this error. 

This is what I added the npmrc for

**Comment by ayezinzu** [comment]
> > No updates to `.npmrc` are required... Please get rid of it and the `pnpm-lock.yaml` updates. Just run `pnpm install`
> 
> image
> 
> @alexiglesias93 Doing so causes this error.
> 
> This is what I added the npmrc for

Removed npmrc and updated lockfile. 

This fixes the CI error

@alexiglesias93

**Pull request merged**